### DISCUSSION TITLE
MTEX 6.0.0

### DISCUSSION THREAD

**Discussion by ralfHielscher** [mtex-toolbox/mtex#2255] [Announcements]
We are very proud to announce that MTEX 6.0. has finally released today. It features some deep changes which allows MTEX to deal with three dimensional grain structures in the same powerful way as we are used to it in two dimensions. A complete description of the new features can be found in the [changelog](https://mtex-toolbox.github.io/changelog).

Have a lot fun testing it and reporting bugs :)

Your MTEX team.

**Comment by Dylannnn24** [comment]
Hello there, 
I am really glad to use MTEX 6.0. I was aiming to use the grain size distribution function, though the code failed at the very beginning of finding and loading the .dream3d file. 
i have open the path folder of the dataset and tried the code: 
> fname = fullfile('data.dream3d');
> grains = grain3d.load(fname);
the outcome said: 
error in grains = grain3d.load(fname) and file or folder not found. 

Could you help me with this, please? is this coding error or somthing else?

**Reply by ralfHielscher** [reply]
Hi Dylan,
I'm sorry you file does not load. Do you think it would be possible for you to send us your data - so we can check what is different in you file in comparison to the files we used for testing.
Ralf.

**Comment by vtvivian** [comment]
Hi Ralf, 

Is there a public copy of 'SmallIN100_MeshStats.dream3d' to download (used here: https://mtex-toolbox.github.io/Grains3D.html), or can you send it to me please?
We're struggling to load our .dream3d dataset so I want to figure out what is the issue.

Thanks
Vivian

**Reply by Djazzair** [reply]
Hi Vivian, I'm struggling on the same issue even with MTEX 6.1.beta3, were you able to solve this problem ?

**Comment by vtvivian** [comment]
Sorry I wasn't able to solve this problem. I did find a copy of the small .dream3d file https://github.com/mtex-toolbox/mtex/blob/develop/data/EBSD/SmallIN100_MeshStats.dream3d, which can be opened in hdfview https://www.hdfgroup.org/download-hdfview/ Perhaps you can compare this one to the file you are trying to open?

Thanks
Vivian


________________________________
From: Djazzair
Sent: Friday, May 16, 2025 10:04
To: mtex-toolbox/mtex
Cc: Vivian Tong; Comment
Subject: [EXTERNAL] Re: [mtex-toolbox/mtex] MTEX 6.0.0 (Discussion #2255)

CAUTION: THIS EMAIL ORIGINATED FROM OUTSIDE OF NPL. Do not click links, open attachments or forward unless you have confirmed the sender is genuine.


Hi Vivian, I'm struggling on the same issue even with MTEX 6.2.beta3, were you able to solve this problem ?

—

Reply to this email directly, view it on GitHub<https://github.com/mtex-toolbox/mtex/discussions/2255#discussioncomment-13168456>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AGT3J5U346OLVH4J6TXPXDL26WSZVAVCNFSM6AAAAABRLPIAGWVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTGMJWHA2DKNQ>.

You are receiving this because you commented.

See our research plans for the next 10 years<https://www.npl.co.uk/resources/metrology-research-roadmap> and find out how you can get involved

Visit the NPL website<https://www.npl.co.uk/?utm_source=Email%20Footer&utm_medium=email&utm_campaign=Homepage> and find out how our cutting-edge measurement science has a positive impact in the real world

________________________________
[http://resource.npl.co.uk/images/npl-logo.jpg]<https://www.npl.co.uk/?utm_source=Email%20Footer&utm_medium=email&utm_campaign=Homepage>        Keep in touch<https://www.npl.co.uk/contact> [http://resource.npl.co.uk/images/twitter.jpg] <http://www.twitter.com/npl>  [http://resource.npl.co.uk/images/facebook.jpg] <https://www.facebook.com/npldigital>  [http://resource.npl.co.uk/images/youtube.jpg] <https://www.youtube.com/npldigital>
________________________________
NPL Privacy Policy<https://www.npl.co.uk/privacy-policy/>

If you have received this message in error, please notify us and remove it from your system.
NPL Management Ltd cannot guarantee that the e-mail or any attachments are free from viruses.

NPL Management Ltd is a company registered in England and Wales, number: 2937881
Registered office: National Physical Laboratory | Hampton Road | Teddington, Middlesex | UK | TW11 0LW
